tests/Demo.py

```diff
@@ -263,17 +277,19 @@ def Main(args=None):
 
     try:
         config = pconfig.parse_configuration_files(conffiles, confpaths)
-        json.dumps(config, indent=4)
+        config_json_str = json.dumps(config, indent=4)
     except pconfig.ConfigurationException as e:
-        logger.error(str(e))
+        LOGGER.error(str(e))
         sys.exit(-1)
 
     plogger.setup_loggers(config.get("Logging", {}))
-    sys.stdout = plogger.stream_to_logger(logging.getLogger("STDOUT"), logging.DEBUG)
-    sys.stderr = plogger.stream_to_logger(logging.getLogger("STDERR"), logging.WARN)
+    sys.stdout = plogger.stream_to_logger(logging.getLogger("STDOUT"),
+                                          logging.DEBUG)
+    sys.stderr = plogger.stream_to_logger(logging.getLogger("STDERR"),
+                                          logging.WARN)
 
     ParseCommandLine(config, remainder)
-    LocalMain(config)
+    local_main(config)
```

Comment:
Do we need to change ParseCommandLine -> parse_command_line

Reply:
Done. I fixed pycodestyle issues, but not all pylint issues.
That will need to be done later.